### PR TITLE
OpenRC build: merge world

### PR DESCRIPTION
**Pull request by kuraga**
`/scripts/systemd.sh` [contains](https://github.com/OndrejHome/gentoo-build/blob/b35557755f5f05e99fec0c26284b811406762045/scripts/systemd.sh#L33) `emerge -vDN @world`.

`/scripts/openrc.sh` [doesn't contain](https://github.com/OndrejHome/gentoo-build/blob/b35557755f5f05e99fec0c26284b811406762045/scripts/openrc.sh).

@OndrejHome , is it correct? Thanks!

### CODE REVIEW

**Comment by OndrejHome** [comment]
Hello @kuraga , It might be OK if the check on previous lines (26-29) succeeds as then it would be never called in `systemd.sh`, therefore it might not be needed in `openrc.sh`.

https://github.com/OndrejHome/gentoo-build/blob/master/scripts/systemd.sh#L26-L29

Have you seen any issue when the `emerge -vDN @world` was not present in `openrc.sh`?

I plan to build new images (both systemd and openrc) in upcoming weeks so I will look closer if that `emerge -vDN @world` is called on recent version of Gentoo.

**Comment by kuraga** [comment]
@OndrejHome ,

firstly, thanks very much for reply!

I'm a newbie in Gentoo **compiling** (I use [Calculate Linux](https://www.calculate-linux.org/)), so this is more of a question than a statement. I just don't understand - how to not **Build The World** :smile: Seriously, which lines compile the `@system`'s and `@world`'s packages?

Could you, please, turn on issues on this repository? Thanks!

**Comment by OndrejHome** [comment]
Hi @kuraga, when doing installation of gentoo you don't need to rebuild `@system` nor `@world` - this because initially you download these packages prebuild in stage3.tar.xz. You can rebuild them later to possibly get optimizations for your hardware.

I have disabled Issues for this repository as it is only fork from author that created these scripts for automated installation that happens to be useful for my otger project. I would not recomment these scripts for first time Gentoo installation or as "installer for Gentoo". 

In short: these scripts "partitions disks, make filesystems, unpack stage3, install few extra packages and compile kernel" on target machine where they are run. They don't rebuild all packages.

Does above make it a bit clearer now?

**Comment by kuraga** [comment]
Yes, it's clear. But no, I don't want to use it as installation. I want to research Gentoo building process. Just it's boring to follow [Gentoo Handbook](https://wiki.gentoo.org/wiki/Handbook) for me :smile: , so I would to use (i.e. understand it, improve for myself, etc.; sure, matching with Handbook, in parallel) a script. For instance, to build a Gentoo version for a container system.

I just thought that `stage3` is used for **building** The World (as such as The Kernel) but now I see it was a silly version: we need build tools for building, thus `stage2`. Just was a dissonance for me. "If we compile the kernel we also would compile packages", I thought.

The correct version (https://wiki.gentoo.org/wiki/Stage_tarball#Stage_3):

> Stage 3 tarballs are compiled from stage 2 tarballs, but contain a system set. Portage includes a quick reference to this set of base packages by using the `@system` set operator.
> 
> Downloading and decompressing a stage 3 tarball as described in Gentoo Handbook will install an almost-complete and almost-functional system (the most important parts still missing are a kernel and a bootloader).

@OndrejHome , big thanks for your patience and teacher skills! I'll try to not disturb you! :smile: Thanks!
